i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/manage-users/README.md

```diff
@@ -8,7 +8,7 @@ sidebar_position: 6
 
 在它们的帮助下，可以执行以下操作：
 
-- 按全名、电话或电子邮件搜索用户，并查看他们的个人资料，包括用户名、头像、社交关系等。
+- 按全名、电话或邮箱搜索用户，并查看他们的个人资料，包括用户名、头像、社交关系等。
```

Comment:
also ID and username?

Reply:
The current implementation does not support search `id` and `username` for now.

Reply:
See Slack thread [discussion](https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1656761171563819)